org/diceresearch/nebula/utils/notification.py

```diff
@@ -0,0 +1,40 @@
+import firebase_admin
+from firebase_admin import credentials, messaging
+
+
+# Initialize Firebase
+cred = credentials.Certificate('../resources/nebula-dev-c8f4a-firebase-adminsdk-acvxz-db4b4471d0.json')
```

Comment:
Can we make this path configurable? Other people may have other file names for their credentials :wink: 

I would also like to make the firebase thing optional, i.e., if the credentials are not set, firebase will not be initialized. This makes it easier to test / run the software without the firebase credentials :slightly_smiling_face:

Reply:
Maybe we could move this path to [settings.py](https://github.com/dice-group/NEBULA/blob/main/org/diceresearch/nebula/settings.py)? It's where we currently keep most of the configuration.